dive_core/common/emulate_pm4.cpp

```diff
@@ -822,7 +822,7 @@ bool EmulatePM4::AdvanceCb(const IMemoryManager &mem_manager,
     // When it shows up in production driver captures, the IB addr and size values are all
     // garbage. Not sure what it's used for at this point.
     if (type7_header.opcode == CP_COND_INDIRECT_BUFFER_PFE)
-        DIVE_LOG("Packet ignored: CP_COND_INDIRECT_BUFFER_PFE\n");
+        DIVE_DEBUG_LOG("Packet ignored: CP_COND_INDIRECT_BUFFER_PFE\n");
```

Comment:
use debug log so that the message is disabled in release build

Reply:
Sounds good!